deploy/scripts/Create-IomtWebJobsEnvironment.ps1

```diff
@@ -0,0 +1,109 @@
+<#
+.SYNOPSIS
+Creates a new IoMT FHIR Connector for Azure without using Stream Analytics
+.DESCRIPTION
+#>
+param
+(
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateLength(5,12)]
+	[ValidateScript({
+        if ("$_" -cmatch "(^([a-z]|\d)+$)") {
+            return $true
+        }
+        else {
+			throw "Environment name must be lowercase and numbers"
+            return $false
+        }
+    })]
+    [string]$EnvironmentName,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateSet('Australia East','East US','East US 2', 'West US', 'West US 2','North Central US','South Central US','Southeast Asia','North Europe','West Europe','UK West','UK South')]
+    [string]$EnvironmentLocation = "North Central US",
+
+    [Parameter(Mandatory = $false)]
+    [ValidateSet('R4')]
+    [string]$FhirVersion = "R4",
+
+    [Parameter(Mandatory = $false)]
+    [string]$SourceRepository = "https://github.com/microsoft/iomt-fhir",
+  
+    [Parameter(Mandatory = $false)]
+    [string]$SourceRevision = "master",
+
+    [Parameter(Mandatory = $true)]
+    [string]$FhirServiceUrl,
+
+    [Parameter(Mandatory = $true)]
+    [string]$FhirServiceAudience,
+
+    [Parameter(Mandatory = $false)]
+    [string]$EnvironmentDeploy = $true
+)
+
+Function BuildPackage() {
+    try {
+        Push-Location $currentPath
+        cd ../../src/console/
```

Comment:
Just a question: do we assume that the script is being executed at a specific location, like "deploy/scripts/"? is that possible to set a working directory with ```dotnet```?

Reply:
I would be better to just resolve the path and build against it if possible.  Changing directory and keeping target can get tricky.

---
In reply to: [544493385](https://github.com/microsoft/iomt-fhir/pull/83#discussion_r544493385) [](ancestors = 544493385)